kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -4045,9 +4045,11 @@ public boolean onItemClick(int position, MenuItem item) {
                     attachReplyCancelLayout.setVisibility(VISIBLE);
                     replayRelativeLayout.setVisibility(VISIBLE);
                 } catch (Exception e) {
-
+                    e.printStackTrace();
```

Comment:
@shubhamtewari @reytum do we use  e.printStackTrace(); at other places in the sdk too? or we use only log statement

Reply:
on searching for printStackTrace, I found 100+ occurences. They were usually inside the catch block. Log is generally used on other cases.